webview/src/experiments/components/table/Cell.tsx

```diff
@@ -44,6 +46,16 @@ export const FirstCell: React.FC<
           isPlaceholder && styles.groupPlaceholder
         )
       })}
+      tabIndex={0}
+      aria-checked={bulletColor !== undefined}
+      role="switch"
+      {...clickAndEnterProps(toggleExperiment)}
+      // onClick={toggleExperiment}
+      // onKeyDown={e => {
+      //   if (e.key === 'Enter' || e.key === ' ') {
+      //     toggleExperiment(e)
+      //   }
+      // }}
```

Comment:
Don't forget to remove the commented out code.

Reply:
Weird, this should have been caught by the linter